server/gradlew.bat

```diff
@@ -0,0 +1,89 @@
+@rem

```

Comment:
I think only the root folder should have this .bat included. (Unless I am wrong ofc 🙃 I don't seem to be able to find another repository where this is in the module folders)

Reply:
I was trying to do `gradle init` to convert "pom.xml", but I didn't remove all of the wrapper.

Reply:
Removed